tests/integration/dotnet/basic/Program.cs

```diff
@@ -33,6 +35,13 @@ static Task<int> Main(string[] args)
                 },
             });
 
+            // Test that JSON data marhalling works.
+            var revision = new ControllerRevision("rev", new ControllerRevisionArgs
+            {
+                Data = JsonDocument.Parse("{\"foo\":42}"),
```

Comment:
note: I also support you just writin: `Data = @"{""foo"":42}"` to make it slightly less verbose.

Reply:
I think making it clear this is JSON (and will be returned in outputs as JSON)  is worth it here.